src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -33,29 +33,20 @@
   private final Item item;
   private final User user;
   private final User proxy;
+  private final DateTime oldDueDate;
```

Comment:
Maybe we could call this `originalDueDate` to reflect that it was the due date that the loan had when the current process started?

Reply:
Makes sense, I'll do this change.

Reply:
Done